httpbin/cmd/cmd_test.go

```diff
@@ -472,6 +527,7 @@ func TestMainImpl(t *testing.T) {
 		getHostname func() (string, error)
 		wantCode    int
 		wantOut     string
+		regex       bool
```

Comment:
I added this one for `server error` and `tls cert error` tests in the **TestMainImpl** because in these tests were failing because I was unable to mock slog without giving slog as input to `mainImpl` function. Instead of updating function it made sense for me to just check these two tests using regex.

Reply:
Yeah, this kind of test is going to be fairly fiddly to write with these logging changes.  Unfortunately, [as noted below](https://github.com/mccutchen/go-httpbin/pull/179#discussion_r1734660515), this approach doesn't actually work.